pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -56,7 +56,14 @@ import (
 )
 
 // ReconcileTime is the delay between reconciliations
-const ReconcileTime = 30 * time.Second
+const (
+	ReconcileTime = 30 * time.Second
+	Node = "node"
+	Controller = "controller"
+)
+
+var ds_restarted_key = ""
+var ds_restarted_value = ""
```

Comment:
What is `ds`?

Reply:
daemonSet

Reply:
ohhh
I thought it's "data structure".
Please rename it accordingly.